gov2/workflows/user_pools_and_lambda_triggers/actions/cloud_formation_actions.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package actions
+
+import (
+	"context"
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudformation"
+)
+
+// snippet-start:[gov2.cloudformation.CloudFormationActions.complete]
+
+// StackOutputs defines structured data for the outputs from a specific stack.
+type StackOutputs struct {
```

Comment:
We already have all this information in the DescribeStacks return value, so we shouldn't need to re-declare everything

Reply:
fixed